heuristic_metrics.json

```diff
@@ -1 +1 @@
-{"commits_train": 50000, "bugs_fraction": 0.5941, "nonbugs_fraction": 0.4059, "n_labeling_functions": 352, "coverage_train": 0.99416, "coverage_test": 0.7426, "majority_accuracy_train": 0.827, "majority_accuracy_test": 0.7204}
\ No newline at end of file
+{"n_labeling_functions": 302, "coverage_train": 0.61467, "coverage_test": 0.7152818991097922, "majority_accuracy_test": 0.7114243323442137}
```

Comment:
Have 50 functions gotten removed as a result of refactoring?

Reply:
After I reran `apply_heuristics` train coverage dropped to 24%, is that related?

Reply:
Hmm, odd. What train set did you use? These values are what I get after the removal.

Reply:
oh, sorry I was running on `a_a` training set, now I am running on `b_b` (which you ran, right?) and the numbers are around the same as yours. 

The difference is quite significant between 24% and 61% though. I would expect it to be less.